lib/widgets/place_picker.dart

```diff
@@ -236,7 +230,7 @@ class PlacePickerState extends State<PlacePicker> {
 
     previousSearchTerm = place;
 
-    if (context == null) {
+    if (context.mounted) {
```

Comment:
Been a long time I did Flutter, so I'm unsure.

Shouldn't this be:

```suggestion
    if (!context.mounted) {
```

Reply:
sorry i forgot to place ! with condition. i am changing it.